src/sentry/api/serializers/models/event.py

```diff
@@ -221,3 +221,44 @@ def serialize(self, obj, attrs, user):
         result['entries'] = [e for e in result['entries']
                              if e['type'] != 'breadcrumbs']
         return result
+
+
+class SnubaEvent(object):
+    """
+        A simple wrapper class on a row (dict) returned from snuba representing
+        an event. Provides a class name to register a serializer against, and
+        Makes keys accessible as attributes.
+    """
+
+    # The list of columns that we should request from snuba to be able to fill
+    # out a proper event object.
+    selected_columns = [
+        'event_id',
+        'message',
+        'user_id',
+        'username',
+        'email',
+    ]
+
+    def __init__(self, kv):
+        self.__dict__ = kv
+
+
+@register(SnubaEvent)
+class SnubaEventSerializer(Serializer):
+    """
+        A bare-bones version of EventSerializer which uses snuba event rows as
+        the source data but attempts to produce a compatible (subset) of the
+        serialization returned by EventSerializer.
+    """
+    def serialize(self, obj, attrs, user):
+        return {
+            'id': 0,  # This is not relevant for snuba-sourced events, but some things expect it
```

Comment:
A little concerned about the front end using this for a React component key or store ID or something and wreaking havok unintentionally.

Reply:
Yeah I think I can actually remove it. The tests initially depended on `id` but then I switched them to using `event_id` instead. 

I will have to change the react stuff as well though, to link to the events based on the `event_id` which I think might mean using those same search links that we used for the hack week project. Or maybe its worth putting in the effort to make a URL for the individual event page that supports accessing events by their event_id directly